x-pack/plugins/code/public/style/_layout.scss

```diff
@@ -0,0 +1,179 @@
+.euiFlexItem.codeContainer__adminWrapper {
+  padding: $euiSize $euiSizeXL;
+  flex-grow: 1;
+}
+
+.codeContainer__root {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.codeContainer__root--inner {
+  display: flex;
+  flex-direction: row;
+  flex-grow: 1;
+  height: 100%;
+}
+
+.codeContainer__main {
+  width: calc(100% - 16rem);
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.codeContainer__import {
+  max-width: 56rem;
+  margin: auto;
+}
+
+.codeContainer__setup {
+  width: 56rem;
+  margin: 0 auto;
+  padding: 0 0 $euiSizeXXL 0;
+
+  .codeContainer__setup--step {
+    margin-top: $euiSizeXXL;
+    width: 100%;
+  }
+}
+
+.codeContainer__editor {
+  overflow: hidden;
+  flex-grow: 1;
+}
+
+.codeContainer__blame {
+  position: relative;
+  max-height: calc(100% - (6rem + 1px));
+  flex-grow: 1;
+}
+
+.codeContainer__directoryView, .codeContainer__history {
+  flex-grow: 1;
+  overflow: auto;
+}
+
+.codeContainer__searchBar {
+  width: 100%;
+}
+
+.codeContainer__select {
+  margin-right: $euiSizeS;
+}
+
+.codeContainer__tabs {
+  height: 3.5rem;
+}
+
+.codeContainer__search--inner {
+  overflow-y: scroll;
+  padding: 1rem;
+}
+
+.codeContainer__search {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+}
+
+.codeContainer__symbolTree {
+  padding: $euiSizeL $euiSizeM;
+  position: relative;
+  display: inline-block;
+  min-width: 100%;
+  height: 100%;
+}
+
+.codeContainer__search--results {
+  margin-top: $euiSize;
+}
+
+.codeSidebar__container {
+  background-color: $euiColorLightestShade;
+  border-right: solid 1px $euiBorderColor;
+  flex-grow: 1;
+  flex-shrink: 1;
+  overflow: auto;
+}
+
+.codeSearchbar__container {
+  height: 3.5rem;
+  padding: $euiSizeS;
+  border-bottom: $euiBorderWidthThin solid $euiBorderColor;
+}
+
+.code-search-settings-flyout {
```

Comment:
nit: this class's naming is not quite aligned with the others though.

Reply:
And then yes, this should be renamed to follow suit. `codeSearchSettings__flyout` based on the other naming in this file.